CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 All notable changes to this project will be documented in this file.
 
+## [v5.3.0](https://github.com/voxpupuli/puppetboard/tree/v5.3.0) (2024-02-07)
+
+[Full Changelog](https://github.com/voxpupuli/puppetboard/compare/v5.2.0...v5.3.0)
+
+**Implemented enhancements:**
+
+- Add favicon to radiator [\#1023](https://github.com/voxpupuli/puppetboard/pull/1023) ([surdaft](https://github.com/surdaft))
+
+**Merged pull requests:**
+
+- Bump markupsafe from 2.1.3 to 2.1.5 [\#1022](https://github.com/voxpupuli/puppetboard/pull/1022) ([dependabot[bot]](https://github.com/apps/dependabot))
```

Comment:
didn't we ignore dependabot bumps in our config in other repos? not blocking, but seems rather noisy

Reply:
we ignore dependabot bumps *for actions* only at the moment.